### PR TITLE
Create NVidia.ru.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

**Comment by koops76** [comment]
#3069

File: src/chrome/content/rules/NVidia.ru.xml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	<target host="shield.nvidia.ru" />
+	<target host="store.nvidia.ru" />
+	<target host="support-shield.nvidia.ru" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://(www\.)?nvidia\.ru/shield/" to="https://www.nvidia.ru/shield/" />
+
+	<test url="http://nvidia.ru/shield/" />
+	<test url="http://www.nvidia.ru/shield/" />
```
Comment (J0WI) [inline src/chrome/content/rules/NVidia.ru.xml]: Please add an exclusion for `^http://nvidia\.ru/$` and `^http://www\.nvidia\.ru/$` and remove it from the whitelist.

File: src/chrome/content/rules/NVidia.ru.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<!--
+	For other NVidia coverage, see NVidia.xml.
```
Comment (J0WI) [inline src/chrome/content/rules/NVidia.ru.xml]: Need to be added to the list in `NVidia.xml`

File: src/chrome/content/rules/NVidia.ru.xml (hunk 1 above)
Comment (koops76) [inline src/chrome/content/rules/NVidia.ru.xml]: @J0WI Done.

File: src/chrome/content/rules/NVidia.ru.xml (hunk 2 above)
Comment (koops76) [inline src/chrome/content/rules/NVidia.ru.xml]: @J0WI Done.

**Pull request merged**